.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+        node:
+          - 14
```

Comment:
I believe we wanted to test against the default Node version that ships with Splunk as well which appears to be `v8.17.0`, could you add that to the testing matrix or remind me of any testing issues that we may have run into while testing against node 8?

Reply:
Updated matrix.